### DISCUSSION TITLE
rAudio 1 • 20211121: Bookmarks no longer work

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#641] [Fixed Issues]
@rern 
What's new with bookmark handling? Existing bookmarks no longer work, same for new bookmarks. Either case I get:
```
Network storage not available
To setup:
Settings > System - Storage -+
```
But storage (NAS) is available.

Detail:
Contents of an old bookmark file from backup:
`NAS/Musik/Jazz`

Contents of corresponding new bookmark file:
```
NAS/Musik/Jazz
/mnt/MPD/NAS/Musik/Jazz/coverart.jpg
```

**Comment by rern** [comment]
Try the UPDATE branch.

**Comment by klassikmann** [comment]
Same situation.

**Reply by klassikmann** [reply]
Solved in rAudio 1 • 20211122
